lib/alerts_viewer_web/live/bus_live.html.heex

```diff
@@ -59,14 +59,18 @@
   <:col :let={route} label="Name">
     <%= Route.name(route) %>
   </:col>
-  <:col :let={route} label="Individual Vehicle Schedule Adherence (seconds)">
-    <%= @stats_by_route |> RouteStats.vehicles_schedule_adherence_secs(route) |> Enum.join(", ") %>
+  <:col :let={route} label="Individual Vehicle Schedule Adherence (minutes)">
+    <%= @stats_by_route
+    |> RouteStats.vehicles_schedule_adherence_secs(route)
+    |> Enum.map(&seconds_to_minutes(&1))
```

Comment:
comment: What you have here is totally fine, but just to point out that since there are no other arguments, this can also be written: `|> Enum.map(&seconds_to_minutes/1)`